frontend/components/puzzleA.jsx

```diff
@@ -5,14 +5,15 @@ import { Narration } from '../wrappers';
 import { isSolved } from '../helpers';
 
 const SafeImage = styled.img`
+  display: inline-flex;
   height: 150px;
   width: 75px;
   margin-right: auto;
   margin-left: auto;
 `;
 const Button = styled.button`
-  margin-right: 5px;
-  ${props => props.selected && 'border-style: inset;'}
+  margin-right: 2px;
+  ${props => props.selected ? 'border-style: inset; border-color: #7C9790; border-width: 5px' : 'background-color: transparent; border-color: transparent'}
```

Comment:
i agree this is more obvious, but if we're no longer using the button look, we can probably take away the `border-style: inset` piece and just show a border when something is selected. we may as well stop using a button on L14 as well

Reply:
What is L14?

Reply:
Line 14, `const Button = styled.button`

Reply:
Not sure I understand what you are saying. Are you saying just to remove the border-style: inset from the line?

Reply:
yea, and stop using a button, since it comes with default styling. just use a div instead and we can make it look however we want